lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""
+Script to config AGW in stateful and stateless mode
+"""
+
+import argparse
+import os
+import subprocess
+import sys
+import time
+
+from enum import Enum
+
+from magma.configuration.service_configs import (
+    load_override_config,
+    load_service_config,
+    save_override_config,
+)
+
+return_codes = Enum(
+    "return_codes", "STATELESS STATEFUL CORRUPT INVALID", start=0
+)
+SERVICE_CONFIG_NAMES = [
+    ("mme", "use_stateless"),
+    ("mobilityd", "persist_to_redis"),
+    ("pipelined", "clean_restart"),
+    ("sessiond", "support_stateless"),
+]
+
+
+def _check_stateless_service_config(service, config_name):
+    config_value = True
+    if service == "pipelined":
+        config_value = False  # pipelined uses inverse logic
```

Comment:
I would prefer passing expected configuration value for stateless as a named parameter with default value True.